tests/mocha/field_dropdown_test.js

```diff
@@ -7,159 +7,110 @@
 suite('Dropdown Fields', function() {
   setup(function() {
     sharedTestSetup.call(this);
+
+    // Invalid value test are expected to log errors.
+    const nativeConsoleError = console.error;
+    this.nativeConsoleError = nativeConsoleError;
+    console.error = function(msg) {
+      if (!msg.includes('Each FieldDropdown option')) {
+        nativeConsoleError.call(this, ...arguments);
+      }
```

Comment:
I'm not sure I understand this part. Are we just hiding errors that have to do with field dropdowns?

Reply:
Yes, it only ignores `console.error` calls containing the string `'Each FieldDropdown option'`, which happens when invalid values are passed to constructor. Any other `console.error` call is called normally (to help us with debugging).